### ISSUE TITLE
Change eauth as a setting

### ISSUE THREAD

**Issue by Diaoul** [tinyclues/saltpad#132]
I want to change eauth to my personnal external_auth plugin for salt, can you make [this](https://github.com/tinyclues/saltpad/blob/e8d6a5265e115a43b40dc17321193f7cbfe26ce5/app/login/AuthService.js#L10) configurable through settings?

**Comment by Lothiraldan** [comment]
Yes of course! It was configurable in the v1 version but I forget to implement it on v2 version, I will make the commit right now.

**Comment by Lothiraldan** [comment]
It is now configurable on master version, could you give it a try? I would like to finish the support for rest_cherrypy before releasing another version.
